www/index.html

```diff
@@ -11,10 +11,11 @@
         Home page : http://www.kiwix.org
 
         Main authors of this application :
-        Mossroy - mossroy@mossroy.fr
+        Mossroy - https://github.com/mossroy - mossroy@mossroy.fr
         Peter-x - https://github.com/peter-x
+        Jaifroid - https://github.com/Jaifroid
```

Comment:
Thank you! Very kind of you to add me!

Reply:
That's normal : I should have done that earlier.
Thanks again for all the work you do on this project.